src/data/add_map.py

```diff
@@ -326,15 +310,8 @@ def add_int_features(int_lines, dir1, dir2, featlist):
     print("Adding features: " + ','.join(feats))
     get_mapping(non_ints_with_candidates, feats)
 
-    output = [{
-        'geometry': geojson.LineString([y for y in x['line'].coords]),
-        'properties': x['properties']} for x in non_ints_with_candidates]
-    output = util.prepare_geojson(output)
-
-    # Write the non-intersections segments back out with the new features
-    with open(os.path.join(
-            MAP_FP, 'non_inters_segments.geojson'), 'w') as outfile:
-        geojson.dump(output, outfile)
+    non_inters = [Segment(x['line'], x['properties'])
```

Comment:
Is this stuff all from your creation of the Segment class? Did the removal of week unlock this somehow? Just generally curious.

Reply:
It's just a little cleanup. I had previously calculated the intersection features and written them to inters_segments.geojson/inter_and_non_int.geojson  in create_segments, but I didn't ever get around to updating the features that were added via add_map, nor did I use the features from the geojson files - they were still being calculated again in make_canon_dataset. I wanted to draw from the geojson files in make_canon_dataset since I was already in there, which meant I had to update the maps, and it was just a little cleaner to use Segment objects, so I could use util.write_segments (also created in this PR, but moved from create_segments.py to util.py since I'd be using it more than once) to update the geojson files.